krokedil/klarna-payments-for-woocommerce
# Register WooCommerce navigation items

* Adds the menu items for the new WC nav experience.
* Fixes some issues with the wrap on WC embed pages.

### Screenshots
<img width="1047" alt="Screen Shot 2020-11-12 at 4 02 09 PM" src="https://user-images.githubusercontent.com/10561050/98996176-6f273400-2500-11eb-86ab-b8e1efbef07f.png">


### Testing

1. Clone and install/activate https://github.com/woocommerce/woocommerce-admin
1. Enable the navigation feature at `wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
1. Check that the "Klarna" extension is registered and menu items work as expected.
1. Check that when the navigation is disabled, styling and padding remains consistent.
